assets/css/styles.css

```diff
@@ -807,12 +824,58 @@ input:focus + .input-label {
         padding: 8px 0 16px 0;
     }
 
-    nav > ul > li {
+    /* nav > ul > li {
         margin: 0 5px;
-    }
+    } */
 
     .project-text {
         padding: 16px 0 0 0;
     }
 }
 
+@media only screen and (max-width: 770px) {
+    .eportfolio {
+        display: none;
+    }
+
+    .logo .name {
+        font-size: 26px;
+    }
+
+    .cert-full {
+        display: none;
+    }
+}
+
+/*---------------------------------------- PROJECTS.HTML styling ----------------------------------------*/
+
+
+.certificates-line-title {
```

Comment:
Pretty much a copy-paste from the projects styling, but had to change the `before` and `after` that add the pretty line on both sides, since it would overlap with the text.